operationsgateway_api/logging.ini.example

```diff
@@ -0,0 +1,38 @@
+[loggers]
+keys=root,uvicorn.access
+
+[handlers]
+keys=fileHandler,consoleHandler
+
+[formatters]
+keys=fileFormatter,consoleFormatter
+
+[logger_root]
+level=DEBUG
+handlers=fileHandler,consoleHandler
+qualname=root
+propagate=0
+
+[logger_uvicorn.access]
+level=INFO
+handlers=fileHandler,consoleHandler
+qualname=uvicorn.access
+propagate=0
+
+[handler_fileHandler]
+class=logging.handlers.TimedRotatingFileHandler
+formatter=fileFormatter
+args=('logs.log', 'D', 1, 20,)
```

Comment:
Why is the log file called logs.log? Can it be called operationsgateway-api.log like it seems to be on my development setup? Or is this following some kind of convention?